src/clarity-angular/dropdown/_dropdown.clarity.scss

```diff
@@ -144,6 +132,24 @@ $dropdown-white: clr-getColor(lightest);
                 text-decoration: none;
             }
 
+            &.expandable {
+                $clr-dropdown-caret-xOffset: baselineRem(8/24);
```

Comment:
This one is not in the spec per se.. but it's used for placing the caret for the expandable items (items with submenu). I'll still change it to use the `$clr_baselineRem_1` here.